main.yml

```diff
@@ -756,6 +756,9 @@ application-services:
       - id: service-registry
         title: "Service Registry"
         group: overview
+      - id: smart-events
```

Comment:
main.yml is only used to upload assets to Akamai. It no longer handles app location or navigation. You have to register a new FE module. Please follow these steps: https://github.com/RedHatInsights/cloud-services-config#chromefed-modulesjson

Reply:
@Hyperkid123 I've added the navigation link to `application-services-navigation.json`